tlslite/keyexchange.py

```diff
@@ -229,40 +229,41 @@ def makeClientKeyExchange(self):
         clientKeyExchange.createRSA(self.encPremasterSecret)
         return clientKeyExchange
 
-# the DHE_RSA part comes from IETF ciphersuite names, we want to keep it
-#pylint: disable = invalid-name
-class DHE_RSAKeyExchange(KeyExchange):
-    """
-    Handling of ephemeral Diffe-Hellman Key exchange
 
+class ADHKeyExchange(KeyExchange):
+    """
+    Handling of anonymous Diffe-Hellman Key exchange
+    FFDHE without signing serverKeyExchange useful for anonymous DH
     NOT stable API, do NOT use
     """
 
     def __init__(self, cipherSuite, clientHello, serverHello, privateKey):
```

Comment:
`def __init__(self, cipherSuite, clientHello, serverHello):` at ADH key exchange and then DHE_RSA constructor will be 
`def __init__(self, cipherSuite, clientHello, serverHello, privateKey):
        super(ADHKeyExchange, self).__init__(cipherSuite, clientHello, serverHello, privateKey)`, yes? 
And I was looking to `tlsconnection.py` and `def _serverAnonKeyExchange(self, clientHello, serverHello, keyExchange, cipherSuite, settings)` doesnt use `clientHello` and `settings` (before and after move ADH to `keyexchange.py`), dunno if I didnt notice use of them, but I think, that they shouldn't be here, true?